lib/src/interfaces/Options.ts

```diff
@@ -1138,6 +1138,13 @@ export interface NavigationBarOptions {
   visible?: boolean;
 }
 
+/**
+ * Configure iOS' default background color of View
+ */
```

Comment:
The comment should be more generalised as `WindowOptions` can potentially include more than just the background color.